src/practice/ts/components/StoreHistory/HistoryStore.ts

```diff
@@ -0,0 +1,41 @@
+interface IHistroy {
+  url: string;
+  timestamp: Date;
+  title: string;
+}
+
+interface IHistories {
+  [id: string]: IHistroy;
+}
+
+interface ISetHistoryArgs {
+  id: number;
+  url: string;
+  title: string;
+}
+
+class HistoryList {
```

Comment:
파일명과 동일하게 class 명을 변경해도 좋을 것 같아요.
```suggestion
class HistoryStore {
```

Reply:
말씀하신대로 수정하는게 훨씬 좋을 것 같네요